examples/2-using-subscriptions/README.md

```diff
@@ -1,21 +1,35 @@
 ## About
 
-This is a basic example of subscriptions being used with URQL.
+This is a basic example of subscriptions being used with `urql`. It demonstrates some of the core principles around wiring up subscriptions, including:
```

Comment:
We need to standardize the casing - I vote "Urql". Capitalized (proper noun) and not as spammy/agressive as URQL.

Not something for this pr, but worth us thinking about.

Reply:
In all of the markdown of our docs we've now gone with "urql" (also when possible stylised as monospaced) which was discussed on Slack.

We can reopen that discussion, but frankly I think optically "urql" is fine and works well, "Urql" is OK but would now be an inconsistency given the docs, and "URQL" would be highly inconsistent and also doesn't read rather nicely (definitely shouty)

Reply:
My issue with all lowercase is that there's not much emphasis on the name which leaves it feeling a little deflated. We can highlight the name in markdown which is awesome but in other places such as a tweet, the name urql just gets lost amongst the words.

Sounds like all lowercase is the standard for now so let's go with that. Maybe there are other examples of all lowercase being used for branding? If you find any forward them to me!